wwwroot/index.html

```diff
@@ -24,128 +33,335 @@
 	<script src="libs/js/bootstrap.min.js"></script>
 	<script src="libs/js/jsplumb.js"></script>
 	<script src="libs/js/slick.js"></script>
+	<!-- Schema.org JSON-LD Markup for Search Engine Optimization -->
+	<script type="application/ld+json">
+	{
+		"@context": "https://schema.org",
+		"@type": "Corporation",
+		"name": "zkSNACKs",
+		"url": "https://zksnacks.com",
+		"logo": "https://zksnacks.com/assets/zksnacks.png",
+		"contactPoint": [
+			{
+			"@type": "ContactPoint",
+			"telephone": "",
+			"contactType": "customer service",
+			"email": "info@zksnacks.com"
+			}
+			],
+	"sameAs": [
+		"https://twitter.com/Zksnacks_LTD",
+		"https://www.linkedin.com/company/zksnacks/",
+		"https://github.com/zkSNACKs",
+		"https://www.crunchbase.com/organization/wasabi-wallet"
+	]
```

Comment:
Fix the formatting like this one:
```suggestion
	{
		"@context": "https://schema.org",
		"@type": "Corporation",
		"name": "zkSNACKs",
		"url": "https://zksnacks.com",
		"logo": "https://zksnacks.com/assets/zksnacks.png",
		"contactPoint": [
			{
				"@type": "ContactPoint",
				"telephone": "",
				"contactType": "customer service",
				"email": "info@zksnacks.com"
			}
		],
		"sameAs": [
			"https://twitter.com/Zksnacks_LTD",
			"https://www.linkedin.com/company/zksnacks/",
			"https://github.com/zkSNACKs",
			"https://www.crunchbase.com/organization/wasabi-wallet"
		]
```